app/controllers/home_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
     @favorite_projects = current_user.favorites
     @projects = current_power.projects
     @editable_project_ids = current_power.updatable_projects.pluck(:id)
+    @host_genomes = HostGenome.all
```

Comment:
There are some host genomes with "__test__" in the name which we just use for test samples. Probably shouldn't show those in front-end. You can replace `HostGenome.all` with something like `HostGenome.all.reject {|hg| hg.name.downcase.include?("__test__")}`